resources/widgetlist.ts

```diff
@@ -285,32 +288,33 @@ class WidgetList extends HTMLElement {
     if (!this._connected)
       return;
 
-    this.rootElement.style.width = this._rowcolsize * this._elementwidth;
-    this.rootElement.style.height = this._rowcolsize * this._elementheight;
+    this.rootElement.style.width = String(this._rowcolsize * this._elementwidth);
+    this.rootElement.style.height = String(this._rowcolsize * this._elementheight);
 
     let x = this._xinc1 < 0 ? -this._elementwidth : 0;
     let y = this._yinc1 < 0 ? -this._elementheight : 0;
     let rowcolindex = 0;
     let count = 0;
 
-    for (const i in this._sorted) {
-      const id = this._sorted[i];
-      console.assert(id !== 0, 'An id in _sorted isn\'t in _elements?');
-      const container = this.shadowRoot.getElementById('child' + id);
-      console.assert(container !== null, 'Element with id child' + id + ' is missing?');
+    this._sorted.forEach((id: number) => {
+      if (id === 0)
+        throw new Error('An id in _sorted isn\'t in _elements?');
```

Comment:
Throwing an error would interupt the workflow, may be not intented?

Reply:
`console.assert()` will do the same thing.

Reply:
https://developer.mozilla.org/zh-CN/docs/Web/API/Console/assert
MDN tells me it wouldn't interrupt.

Reply:
The code should interrupt here, some value is null, and can't operate it.